### PR TITLE
Feature/sdev 3256 gene list updates

### PR DESCRIPTION
**Pull request by dustinbleile**
Add get_therapeutic_associated_genes from pori_ipr_python and update tests.

### CODE REVIEW

File: tests/test_match.py [hunk 1]
```diff
@@ -402,6 +402,16 @@ def test_genomic_coordinates(self, conn):
         match.match_positional_variant(conn, genomic)
         # no assert b/c checking for no error rather than the result
 
+    def test_tert_promoter(self, conn):
+        assert match.match_positional_variant(conn, 'TERT:c.-124C>T')
+
```
Comment (dustinbleile) [inline tests/test_match.py:407]: This adds a test for the TERT probe

File: graphkb/genes.py [hunk 2]
```diff
@@ -63,6 +70,43 @@ def get_oncokb_tumour_supressors(conn: GraphKBConnection) -> List[Ontology]:
     return _get_oncokb_gene_list(conn, TUMOUR_SUPPRESSIVE)
 
 
+def get_therapeutic_associated_genes(graphkb_conn: GraphKBConnection) -> List[Ontology]:
+    """Genes related to a cancer-associated statement in Graphkb."""
```
Comment (dustinbleile) [inline graphkb/genes.py:74]: This is from pori_ipr_python, but is easier to understand with the other gene lists in this file.

**Comment by dustinbleile** [comment]
Any comments for the review?

File: tests/test_genes.py [hunk 3]
```diff
@@ -62,3 +64,14 @@ def test_find_fusion_genes(conn, gene):
     result = get_genes_from_variant_types(conn, FUSION_NAMES)
     names = {row['name'] for row in result}
     assert gene in names
+
+
+@pytest.mark.parametrize('gene', CANNONICAL_THERAPY_GENES + CANONICAL_ONCOGENES + CANONICAL_TS)
+def test_get_therapeutic_associated_genes(conn, gene):
```
Comment (mathieulemieux) [inline tests/test_genes.py:70]: Not a blocker, just a question. Is that calling the test as many time as they are genes in the list? Could we instead, at line 77, have something like 'for gene in genes: assert gene in names' ?

**Review by mathieulemieux — DISMISSED** [review]
I ran the test suite with succes on dev.

File: tests/test_genes.py (hunk 3 above)
Comment (dustinbleile) [reply tests/test_genes.py]: I think you are correct and it is simpler, so I've made that change.

**Review by mathieulemieux — APPROVED** [review]

**Pull request merged**